package.json

```diff
@@ -85,7 +85,7 @@
   "devDependencies": {
     "@studio/changes": "^2.0.1",
     "eslint": "^7.13.0",
-    "mocha": "^8.4.0",
+    "mocha": "^9.1.0",
```

Comment:
Not sure if this your intention in this PR, but this is the Mocha version used by the tests in this repo. Mochify itself will still be on version 8 as per the version of mocaccino that is used. Is that the idea here?

Reply:
I wanted to check if mochify.js would be compatible with mocha v9, as per https://github.com/mantoni/mocaccino.js/pull/38#discussion_r774395444.

Reply:
I added another commit to also update mocaccino (temporarily to my fork) to see if there would be any conflicts with mocha.